TorchServe backend pytest suite failing with pytest version.

## Your Environment
pytest version : 6.0.0

## Expected Behavior
The backend pytest suite should pass on the latest pytest version.

## Current Behavior
The pytest suite fails with assertion errors.

## Possible Solution
Use caplog.setlevel() to capture console logs in pytest suite.

## Steps to Reproduce

```
git clone https://github.com/pytorch/serve.git
cd serve
pip install -r requirements/developer.txt
pytest --cov-report html:htmlcov --cov=ts/ ts/tests/unit_tests/;
```

## Failure Logs [if any]
```
(base) USL07109:serve harsh_bafna$ pytest --cov-report html:htmlcov --cov=ts/ ts/tests/unit_tests/;
=========================================================================================== test session starts ============================================================================================
platform darwin -- Python 3.7.7, pytest-6.0.0, py-1.9.0, pluggy-0.13.1
rootdir: /Users/harsh_bafna/serve
plugins: arraydiff-0.3, mock-3.1.0, remotedata-0.3.2, hypothesis-5.8.3, openfiles-0.5.0, doctestplus-0.5.0, astropy-header-0.1.2, cov-2.8.1
collected 35 items                                                                                                                                                                                         

ts/tests/unit_tests/test_beckend_metric.py F                                                                                                                                                         [  2%]
ts/tests/unit_tests/test_model_loader.py .......                                                                                                                                                     [ 22%]
ts/tests/unit_tests/test_model_service_worker.py .............                                                                                                                                       [ 60%]
ts/tests/unit_tests/test_otf_codec_protocol.py .........                                                                                                                                             [ 85%]
ts/tests/unit_tests/test_version.py .                                                                                                                                                                [ 88%]
ts/tests/unit_tests/test_worker_service.py ....                                                                                                                                                      [100%]

================================================================================================= FAILURES =================================================================================================
_______________________________________________________________________________________________ test_metrics _______________________________________________________________________________________________

caplog = <_pytest.logging.LogCaptureFixture object at 0x7fd2c70cdd90>

    def test_metrics(caplog):
        """
        Test if metric classes methods behave as expected
        Also checks global metric service methods
        """
        # Create a batch of request ids
        request_ids = {0: 'abcd', 1: "xyz", 2: "qwerty", 3: "hjshfj"}
        all_req_ids = ','.join(request_ids.values())
        model_name = "dummy model"
    
        # Create a metrics objects
        metrics = MetricsStore(request_ids, model_name)
    
        # Counter tests
        metrics.add_counter('CorrectCounter', 1, 1)
        test_metric = metrics.cache[get_model_key('CorrectCounter', 'count', 'xyz', model_name)]
        assert 'CorrectCounter' == test_metric.name
        metrics.add_counter('CorrectCounter', 1, 1)
        metrics.add_counter('CorrectCounter', 1, 3)
        metrics.add_counter('CorrectCounter', 1)
        test_metric = metrics.cache[get_model_key('CorrectCounter', 'count', all_req_ids, model_name)]
        assert 'CorrectCounter' == test_metric.name
        metrics.add_counter('CorrectCounter', 3)
        test_metric = metrics.cache[get_model_key('CorrectCounter', 'count', 'xyz', model_name)]
        assert test_metric.value == 2
        test_metric = metrics.cache[get_model_key('CorrectCounter', 'count', 'hjshfj', model_name)]
        assert test_metric.value == 1
        test_metric = metrics.cache[get_model_key('CorrectCounter', 'count', all_req_ids, model_name)]
        assert test_metric.value == 4
        # Check what is emitted is correct
        emit_metrics(metrics.store)
    
>       assert "hjshfj" in caplog.text
E       AssertionError: assert 'hjshfj' in ''
E        +  where '' = <_pytest.logging.LogCaptureFixture object at 0x7fd2c70cdd90>.text

ts/tests/unit_tests/test_beckend_metric.py:59: AssertionError

---------- coverage: platform darwin, python 3.7.7-final-0 -----------
Coverage HTML written to dir htmlcov

========================================================================================= short test summary info ==========================================================================================
FAILED ts/tests/unit_tests/test_beckend_metric.py::test_metrics - AssertionError: assert 'hjshfj' in ''
======================================================================================= 1 failed, 34 passed in 0.75s =======================================================================================
```

--- a/ts/tests/unit_tests/test_beckend_metric.py
+++ b/ts/tests/unit_tests/test_beckend_metric.py
@@ -6,8 +6,6 @@ from ts.metrics.dimension import Dimension
 from ts.metrics.metrics_store import MetricsStore
 from ts.service import emit_metrics
 
-logging.basicConfig(stream=sys.stdout, format="%(message)s", level=logging.INFO)
-
 
 def get_model_key(name, unit, req_id, model_name):
     dimensions = list()
@@ -30,6 +28,7 @@ def test_metrics(caplog):
     Also checks global metric service methods
     """
     # Create a batch of request ids
+    caplog.set_level(logging.INFO)
     request_ids = {0: 'abcd', 1: "xyz", 2: "qwerty", 3: "hjshfj"}
     all_req_ids = ','.join(request_ids.values())
     model_name = "dummy model"

--- a/ts/tests/unit_tests/test_worker_service.py
+++ b/ts/tests/unit_tests/test_worker_service.py
@@ -1,14 +1,11 @@
 import logging
 import os
-import sys
 
 import pytest
 
 from ts.context import Context
 from ts.service import Service
 from ts.service import emit_metrics
-
-logging.basicConfig(stream=sys.stdout, format="%(message)s", level=logging.INFO)
 
 
 # noinspection PyClassHasNoInit
@@ -50,6 +47,7 @@ class TestService:
 class TestEmitMetrics:
 
     def test_emit_metrics(self, caplog):
+        caplog.set_level(logging.INFO)
         metrics = {'test_emit_metrics': True}
         emit_metrics(metrics)
         assert "[METRICS]" in caplog.text

